duckduckgo/Android
# Hotfix/5.75.1

Task/Issue URL: https://app.asana.com/0/414730916066338/1199662224112775/f
Tech Design URL: 
CC: 

**Description**:
This PR rolls back changes introduced in https://github.com/duckduckgo/Android/pull/1053/files

This will take us back to https://github.com/duckduckgo/Android/issues/879, but we will create a follow-up task to investigate further the issue when signing in with Facebook in some Websites.


**Steps to test this PR**:
1. Go to pinterest.com
1. Login with Facebook
2. Ensure Pinterest tab remains open


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
